cypress/e2e/home/home.cy.js

```diff
@@ -0,0 +1,131 @@
+const path = require("path");
+const API_URL = Cypress.env('API_URL');
+const CUSTOMERS_URL = `${API_URL}/customers`;
+
+describe('Home Page -', () => {
+  beforeEach(() => {
+    cy.setCookie('cookieConsent', 'accepted');
+    cy.visit('/')
+  })
+
+  it('See welcome message with entered name', () => {
+    cy.get('#name').type('Pedro')
+    cy.get('h2').contains('Hi Pedro!').should('be.visible')
+  })
+
+  context('Customer Details', () => {
+    it.only('It shows and hides the customer address', () => {
+      cy.contains('button', 'View').click()
+      cy.contains('h2', 'Customer Details').should('be.visible')
+      cy.get('button[class*="CustomerDetails_showAddressBtn"]').click()
+      cy.contains('h3', 'Address').should('be.visible')
+      cy.contains('p', '87908 Adkins Islands Apt. 944').should('be.visible')
+      cy.contains('p', 'West Sarah').should('be.visible')
+      cy.contains('p', 'Georgia').should('be.visible')
+      cy.contains('p', '79943').should('be.visible')
+      cy.contains('p', 'United States of America').should('be.visible')
+      cy.get('button[class*="CustomerDetails_hideAddressBtn"]').click()
```

Comment:
```suggestion
      cy.get('button[class*="CustomerDetails_hideAddressBtn"]').click()

```

Ao deixar uma linha em branco entre pré-condições, ações e resultados esperados, fica mais fácil entender o que cada parte do teste é.

Falaremos mais sobre isso no último encontro da _Test Design Masterlclass_, ao abordar o padrão AAA (_Arrange, Act, Assert_).

Reply:
O mesmo vale para outros testes.